fastq_filtrator.py

```diff
@@ -0,0 +1,199 @@
+def fastq_filter(gc_bounds, length_bounds, quality_threshold,
+                 save_filtered, input_fastq, output_file_prefix):
```

Comment:
Также подразумевалось задание значений аргументов по-умолчанию прямо при объявлении функции
```suggestion
def fastq_filter(input_fastq, output_file_prefix, gc_bounds=(0, 100), length_bounds=(0, 2**32), quality_threshold=0,
                 save_filtered=False):
```